gh-runner/Dockerfile

```diff
@@ -82,6 +82,7 @@ RUN apt-get update && \
         libhidapi-dev \
         python3-dev \
         python3-pip \
+        python3-yaml \
```

Comment:
probably not needed here as we're not building anything on self-hosted github runners (just running HW tests - using trunner in venv).

Reply:
Right, it seems to not be needed. Removed